### PR TITLE
Minor revision of the TF Mixed Precision guide

### PR DESCRIPTION
**Pull request by 8bitmp3**
The same version of an old PR https://github.com/tensorflow/docs/pull/1778 (linting). Fresh fork.

PTAL @MarkDaoust @lamberta 

(As before, _"there's a section on a technique for mixed precision training called "loss scaling" (I also read about it here https://arxiv.org/abs/1910.12385). It looks like we can say "a loss scale value" or "a loss scale hyperparameter""_...)

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -2,10 +2,10 @@
 # Last matching pattern takes preecedence.
 
 # Default owners for everything in repo.
-*  @lamberta @MarkDaoust @yashk2810 @anirudh161
+*  @lamberta @MarkDaoust @8bitmp3
```
Comment (lamberta) [inline CODEOWNERS:5]: Did you rebase? This update is already checking in master: https://github.com/tensorflow/docs/blob/master/CODEOWNERS

**Review by lamberta — DISMISSED** [review]

File: CODEOWNERS (hunk 1 above)
Comment (lamberta) [reply CODEOWNERS]: I rebased this PR with master and force pushed to fix this.

**Review by lamberta — APPROVED** [review]
